.github/workflows/marqo-migration.yml

```
@@ -0,0 +1,88 @@
+        if [ "${{ inputs.environment }}" = "dev" ]; then
+          echo "DEV_MARQO_API_BASE_URL=${{ secrets.DEV_MARQO_API_BASE_URL }}" >> $GITHUB_ENV
+          echo "PROD_MARQO_API_BASE_URL=${{ secrets.PROD_MARQO_API_BASE_URL }}" >> $GITHUB_ENV
+        else
+          echo "DEV_MARQO_API_BASE_URL=${{ secrets.DEV_MARQO_API_BASE_URL }}" >> $GITHUB_ENV
+          echo "PROD_MARQO_API_BASE_URL=${{ secrets.PROD_MARQO_API_BASE_URL }}" >> $GITHUB_ENV
+        fi
+
+    - name: Run Migration
+      env:
+        DEV_MARQO_API_BASE_URL: ${{ secrets.DEV_MARQO_API_BASE_URL }}
+        PROD_MARQO_API_BASE_URL: ${{ secrets.PROD_MARQO_API_BASE_URL }}       
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)